kymatio/visuals.py

```diff
@@ -0,0 +1,608 @@
+# -*- coding: utf-8 -*-
+"""Convenience visual methods."""
```

Comment:
This can max be in examples or its own package

Reply:
#718